### PR TITLE
Fix indentation and alphabetization, added missing Love IDE packages

### PR DESCRIPTION
**Pull request by lizenas**
* Fix indentation and alphabetization, added language-glsl and autocomplete-glsl packages which are included in Love IDE but not indicated on the list.

### CODE REVIEW

**Comment by JanWerder** [comment]
Hi, thanks for contribution. You replaced linter-luacheck with linter-luaparse, which seems to be a 404, was that intentional? Otherwise it looks good

**Comment by rm-code** [comment]
@lizenas @JanWerder The indentation isn't correct too. The packages shouldn't be a subpoint of LÖVE-IDE which is a bundle of several packages by one author.

So linter-luacheck, Atom-Launcher and possibly a few others shouldn't be listed as subpoints.

**Comment by lizenas** [comment]
Several included packages were already listed as subpoints of Love-IDE, while other included packages were not, so in the absence of any mention of subpoints in CONTRIBUTING.md, I tried to keep the standard that made the most sense to me.

**Comment by rm-code** [comment]
> I tried to keep the standard that made the most sense to me.
No worries. The packages which currently are subpoints of LOVE-IDE are part of the LOVE-IDE package. 

That's why the other packages shouldn't be subpoints.


In your latest commit linter-luacheck still is missing. It's a different package than luaparse.

File: README.md [hunk 1]
```diff
@@ -259,11 +259,13 @@ A categorized community-driven collection of high-quality, awesome [LÖVE](http:
 * [Atom](https://atom.io/) - A hackable text editor for the 21st Century
 	* [Löve Atom](https://atom.io/packages/love-atom) - A lövely provider for Atom's autocomplete+ package
 	* [Löve IDE](https://atom.io/packages/love-ide) - This package auto-installs several utilities for writing Love2D games in Atom
+		* [Autocomplete GLSL](https://atom.io/packages/autocomplete-glsl) - Adds GLSL autocompletion in Atom
```
Comment (rm-code) [inline README.md]: Shouldn't be a subpoint of LÖVE IDE.

File: README.md [hunk 2]
```diff
@@ -259,11 +259,13 @@ A categorized community-driven collection of high-quality, awesome [LÖVE](http:
 * [Atom](https://atom.io/) - A hackable text editor for the 21st Century
 	* [Löve Atom](https://atom.io/packages/love-atom) - A lövely provider for Atom's autocomplete+ package
 	* [Löve IDE](https://atom.io/packages/love-ide) - This package auto-installs several utilities for writing Love2D games in Atom
+		* [Autocomplete GLSL](https://atom.io/packages/autocomplete-glsl) - Adds GLSL autocompletion in Atom
 		* [Autocomplete Löve](https://atom.io/packages/autocomplete-love) - Auto-complete and snippets for LÖVE
 		* [Hyperclick Löve](https://atom.io/packages/hyperclick-love) - A Hyperclick provider for LÖVE which shows the wiki
-	* [Löve Launcher](https://atom.io/packages/love-launcher) - Launch LÖVE 2D for the current project without having to leave Atom
-	* [language-lua](https://github.com/FireZenk/language-lua) - Lua language support in Atom
-	* [linter-luacheck](https://github.com/AtomLinter/linter-luacheck) - Lint Lua on the fly, using luacheck
```
Comment (rm-code) [inline README.md]: Linter luacheck is missing from your PR.

File: README.md [hunk 3]
```diff
@@ -259,11 +259,13 @@ A categorized community-driven collection of high-quality, awesome [LÖVE](http:
 * [Atom](https://atom.io/) - A hackable text editor for the 21st Century
 	* [Löve Atom](https://atom.io/packages/love-atom) - A lövely provider for Atom's autocomplete+ package
 	* [Löve IDE](https://atom.io/packages/love-ide) - This package auto-installs several utilities for writing Love2D games in Atom
+		* [Autocomplete GLSL](https://atom.io/packages/autocomplete-glsl) - Adds GLSL autocompletion in Atom
 		* [Autocomplete Löve](https://atom.io/packages/autocomplete-love) - Auto-complete and snippets for LÖVE
 		* [Hyperclick Löve](https://atom.io/packages/hyperclick-love) - A Hyperclick provider for LÖVE which shows the wiki
-	* [Löve Launcher](https://atom.io/packages/love-launcher) - Launch LÖVE 2D for the current project without having to leave Atom
-	* [language-lua](https://github.com/FireZenk/language-lua) - Lua language support in Atom
-	* [linter-luacheck](https://github.com/AtomLinter/linter-luacheck) - Lint Lua on the fly, using luacheck
+		* [language-glsl](https://atom.io/packages/language-glsl) - OpenGL Shading Language support in Atom
```
Comment (rm-code) [inline README.md]: Shouldn't be a subpoint of LÖVE IDE.

File: README.md [hunk 4]
```diff
@@ -259,11 +259,13 @@ A categorized community-driven collection of high-quality, awesome [LÖVE](http:
...
 	* [Löve Atom](https://atom.io/packages/love-atom) - A lövely provider for Atom's autocomplete+ package
 	* [Löve IDE](https://atom.io/packages/love-ide) - This package auto-installs several utilities for writing Love2D games in Atom
+		* [Autocomplete GLSL](https://atom.io/packages/autocomplete-glsl) - Adds GLSL autocompletion in Atom
 		* [Autocomplete Löve](https://atom.io/packages/autocomplete-love) - Auto-complete and snippets for LÖVE
 		* [Hyperclick Löve](https://atom.io/packages/hyperclick-love) - A Hyperclick provider for LÖVE which shows the wiki
-	* [Löve Launcher](https://atom.io/packages/love-launcher) - Launch LÖVE 2D for the current project without having to leave Atom
-	* [language-lua](https://github.com/FireZenk/language-lua) - Lua language support in Atom
-	* [linter-luacheck](https://github.com/AtomLinter/linter-luacheck) - Lint Lua on the fly, using luacheck
+		* [language-glsl](https://atom.io/packages/language-glsl) - OpenGL Shading Language support in Atom
+		* [language-lua](https://atom.io/packages/language-lua) - Lua language support in Atom
```
Comment (rm-code) [inline README.md]: Shouldn't be a subpoint of LÖVE IDE.

File: README.md [hunk 5]
```diff
@@ -259,11 +259,13 @@ A categorized community-driven collection of high-quality, awesome [LÖVE](http:
...
+		* [linter-luaparse](https://atom.io/packages/linter-luaparse) - Lua syntax error checking in Atom
```
Comment (rm-code) [inline README.md]: As far as I can tell luaparse isn't part of LOVE-IDE either. The repo hasn't been updated in 2 years. Is it still working with the latest Atom versions?

File: README.md [hunk 6]
```diff
@@ -259,11 +259,13 @@ A categorized community-driven collection of high-quality, awesome [LÖVE](http:
...
+		* [Löve Launcher](https://atom.io/packages/love-launcher) - Launch LÖVE for the current project without having to leave Atom
```
Comment (rm-code) [inline README.md]: Shouldn't be a subpoint of LÖVE IDE.

File: README.md (hunk 5 above)
Comment (lizenas) [inline README.md]: Luaparse is part of LOVE-IDE: https://github.com/rameshvarun/love-ide#lua-syntax-checking-via-linter-luaparse

Comment (rm-code) [inline README.md]: Ah perfect, then it can be added as a subpoint to LOVE-IDE :)

Comment (lizenas) [inline README.md]: Luacheck is probably a better linter though, as the repo page says "Unfortunately, luaparse only reports the first syntax error in a given file."

Comment (rm-code) [inline README.md]: I wouldn't mind keeping it on a list as a subpoint of the LOVE-IDE package.

**Comment by JanWerder** [comment]
Thanks, added it!

**Pull request merged**